packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -399,6 +435,16 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
               }, 0)`}
               className={this._classNames.yAxis}
             />
+            <g
```

Comment:
I see that this additional g tag is added to all charts even if they don't have secondary axis enabled. Lets put this g element inside an if check to render only if secondary scale is defined.